packages/shared/src/components/RadialProgress.module.css

```diff
@@ -15,5 +15,9 @@
     transform-origin: center;
     transition: stroke-dashoffset 0.5s ease-out
       var(--radial-progress-transition-delay);
+
+    &:global(.modal) {
+      stroke: var(--theme-label-primary);
+    }
```

Comment:
A more explicit class name should be used I think. `modal` is pretty vague (at least for me)